Samples/Samples.Cards/Program.cs

```diff
@@ -165,351 +193,495 @@ public async Task<ActionResponse> OnCardAction([Context] AdaptiveCards.ActionAct
+    private static AdaptiveCard CreateProfileCardWithValidation()
+    {
+        return new AdaptiveCard
         {
-            return new AdaptiveCard
+            Schema = "http://adaptivecards.io/schemas/adaptive-card.json",
+            Body = new List<CardElement>
             {
-                Schema = "http://adaptivecards.io/schemas/adaptive-card.json",
-                Body = new List<CardElement>
+                new TextBlock("Profile with Validation")
                 {
-                    new TextBlock("Profile with Validation")
-                    {
-                        Weight = TextWeight.Bolder,
-                        Size = TextSize.Large
-                    },
-                    new NumberInput
-                    {
-                        Id = "age",
-                        Label = "Age",
-                        IsRequired = true,
-                        Min = 0,
-                        Max = 120
-                    },
-                    new TextInput
-                    {
-                        Id = "name",
-                        Label = "Name",
-                        IsRequired = true,
-                        ErrorMessage = "Name is required"
-                    },
-                    new TextInput
-                    {
-                        Id = "location",
-                        Label = "Location"
-                    }
+                    Weight = TextWeight.Bolder,
+                    Size = TextSize.Large
                 },
-                Actions = new List<Microsoft.Teams.Cards.Action>
+                new NumberInput
                 {
-                    new ExecuteAction
-                    {
-                        Title = "Save",
-                        Data = new Union<string, SubmitActionData>(new SubmitActionData { NonSchemaProperties = new Dictionary<string, object?> { { "action", "save_profile" } } }),
-                        AssociatedInputs = AssociatedInputs.Auto
-                    }
+                    Id = "age",
+                    Label = "Age",
+                    IsRequired = true,
+                    Min = 0,
+                    Max = 120
+                },
+                new TextInput
+                {
+                    Id = "name",
+                    Label = "Name",
+                    IsRequired = true,
+                    ErrorMessage = "Name is required"
+                },
+                new TextInput
+                {
+                    Id = "location",
+                    Label = "Location"
                 }
-            };
-        }
+            },
+            Actions = new List<Microsoft.Teams.Cards.Action>
+            {
+                new ExecuteAction
+                {
+                    Title = "Save",
+                    Data = new Union<string, SubmitActionData>(new SubmitActionData { NonSchemaProperties = new Dictionary<string, object?> { { "action", "save_profile" } } }),
```

Comment:
Union is an advanced concept in C# (I'm still learning).. is really required for this sample?